emission/tests/funcTests/TestFunctionTiming.py

```diff
@@ -0,0 +1,95 @@
+# emission/tests/funcTests/TestFunctionTiming.py
+
+import logging
+import time
+from typing import Callable, List
+
+# Import the store_dashboard_time and store_dashboard_error functions
+from emission.storage.decorations.stats_queries import (
```

Comment:
we don't use `from X import Y` in our code; our standard is to use `import X as x` and then use `x.function` in the rest of the code.